Source/DafnyCore/Backends/Dafny/DafnyCodeGenerator.cs

```diff
@@ -896,19 +956,20 @@ private class ExprLvalue : ILvalue {
 
       return new ExprLvalue(
         (DAST.Expression)DAST.Expression.create_Index(source, collKind, Sequence<DAST.Expression>.FromElements(index)),
-        (DAST.AssignLhs)DAST.AssignLhs.create_Index(source, Sequence<DAST.Expression>.FromElements(index))
+        (DAST.AssignLhs)DAST.AssignLhs.create_Index(source, Sequence<DAST.Expression>.FromElements(index)),
+        this
       );
     }
 
     protected override ILvalue MultiSelectLvalue(MultiSelectExpr ll, ConcreteSyntaxTree wr, ConcreteSyntaxTree wStmts) {
-      throwGeneralUnsupported();
+      throwGeneralUnsupported("<i>MultiSelectLValue</i>");
```

Comment:
Why the `<i>`s in general?

Reply:
Because it's something no code generator uses so far (we are not compiling to XML or HTML), so looking for this tags in the generated source code makes it easy to collect missing features in a single regex.